### ISSUE TITLE
Thread safety

### ISSUE THREAD

**Issue by s-nel** [dfabulich/sitemapgen4j#24]
For large sites it is desirable to parallelize sitemap generation. Please make `SitemapGenerator` thread-safe

**Comment by dfabulich** [comment]
Good idea, but i don't have time to do this myself. Pull requests welcome!
